tests/baselines/reference/declarationQuotedMembers.js

```diff
@@ -0,0 +1,62 @@
+//// [declarationQuotedMembers.ts]
+export type Timespans =
+    '1 hour' | '4 hours' | '12 hours' | '1 day' | '3 days' | '1 week' | '1 month';
+const TIMESPANS: {[k in Timespans]: number} = {
+    '1 hour': 1 * 60 * 60 * 1000,
```

Comment:
I'd rather see the smallest test possible, so long as it failed before this PR and succeeds after.